### PR TITLE
API: add History

### PR DESCRIPTION
**Pull request by caugner**
[History](https://developer.mozilla.org/docs/Web/API/History)

### CODE REVIEW

File: api/History.json [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "modify_state": {
+        "__compat": {
+          "description": "<code>pushState</code> and <code>replaceState</code>",
+          "support": {
+            "webview_android": {
+              "version_added": "2.2"
```
Comment (jpmedley) [inline api/History.json]: Change to `true`. Webview follows Chrome numbering not Android numbering. I don't have accurate information that far back.

Comment (caugner) [reply api/History.json]: Thanks for the review, I have just pushed a fix.

File: api/History.json [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+            "version_added": true
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "length": {
```
Comment (connorshea) [inline api/History.json]: All the subfeatures are missing status info, this should probably be fixed :)

Comment (caugner) [reply api/History.json]: Good catch once more, @connorshea. I have added these now (fa90807).

**Pull request merged**